specification/network/resource-manager/readme.md

```diff
@@ -102,6 +103,24 @@ input-file:
   - Microsoft.Network/stable/2024-07-01/vmssNetworkInterface.json
   - Microsoft.Network/stable/2024-07-01/vmssPublicIpAddress.json
   - Microsoft.Network/stable/2024-07-01/webapplicationfirewall.json
+suppressions:
+  - code: PutResponseCodes
+    reason: Required for multiple response codes. Reviewed by ARM team.
```

Comment:
>Reviewed by ARM team. [](http://example.com/codeflow?start=50&length=21)

Can u add the reasoning that was provided to the ARM team who suggested or approved this?

Reply:
Require multiple codes is not a legitimate reason..

Reply:
Pl remove the armchangesrequested label once you provide the justification